.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: "\U0001F41BBug report\U0001F41B"
+about: Create a report to help us improve things
+label: bug
+---
+
+### If you need support for xDrip, PLEASE DO NOT FILE A TICKET HERE
+For support, please post a question to the xDripG5 group at Facebook (https://www.facebook.com/groups/xDripG5) or visit the Gitter room at https://gitter.im/jamorham/xDrip-plus.
+Please also search the existing issues for similar problems before opening a new one.
+
+### Subject of the issue
+A clear and concise description of what the bug is.
+
+### Your environment
+- What version of xDrip (e.g. 0.10.3)
```

Comment:
I think it would help if the example was similar to what it would actually be

Reply:
I've added some xDrip specific example text.